readme.md

```diff
@@ -82,13 +86,32 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ output }) => {
+  // Output is an object of file names to { source, permissions } | string
```

Comment:
I'm still digesting this change but I'm wondering if a symlink should instead be an object like `{ symlink: string }`

Reply:
We could equally well do that, yes. Previously it was its own object too, which we could also do - having `files` and `symlinks`.

Reply:
Oh I like that better, `files` and `symlinks`